src/main/scala/org/allenai/pipeline/examples/MinimalSamplePipeline.scala

```diff
@@ -0,0 +1,68 @@
+package org.allenai.pipeline.examples
+
+import java.io.{FileWriter, PrintWriter, File}
+import java.nio.file.Files
+
+import org.allenai.common.Resource
+import org.allenai.pipeline.ExternalProcess.{OutputFileToken, InputFileToken}
+import org.allenai.pipeline.IoHelpers._
+import org.allenai.pipeline._
+import org.apache.commons.io.FileUtils
+import spray.json.DefaultJsonProtocol._
+
+import scala.io.Source
+import scala.reflect.ClassTag
+
+class MinimalSamplePipeline extends Pipeline {
```

Comment:
This class doesn't need to both inherit from Pipeline and instantiate a Pipeline.  It should just instantiate one.